src/fmu/dataio/providers/objectdata/_tables.py

```diff
@@ -44,7 +44,7 @@ def _derive_index(table_index: list[str] | None, columns: list[str]) -> list[str
         logger.debug("Finding index to include")
         for context, standard_cols in STANDARD_TABLE_INDEX_COLUMNS.items():
             for valid_col in standard_cols:
-                if valid_col in columns:
+                if valid_col in columns and valid_col not in index:
                     index.append(valid_col)
```

Comment:
Oh, is that how it is implemented.. I was sure it was connected to the given `content`! 
We have just been lucky here then that there have not been any overlapping column names between the different dict entries 🙂 
I guess your fix will work for now, until we find on a better way of setting the index 👍

Reply:
I thought it was too, so I thought the duplicated column was some other mistake earlier. Oops. Lucky indeed